nemo/collections/tts/models/hifigan.py

```diff
@@ -190,58 +199,58 @@ def validation_step(self, batch, batch_idx):
         pred_denoised = self._bias_denoise(audio_pred, audio_mel).squeeze(1)
         pred_denoised_mel, _ = self.audio_to_melspec_precessor(pred_denoised, audio_len)
 
-        if self.finetune:
+        if self.input_as_mel:
             gt_mel, gt_mel_len = self.audio_to_melspec_precessor(audio, audio_len)
         audio_pred_mel, _ = self.audio_to_melspec_precessor(audio_pred.squeeze(1), audio_len)
         loss_mel = F.l1_loss(audio_mel, audio_pred_mel)
 
-        self.log("val_loss", loss_mel, prog_bar=True, sync_dist=True)
+        self.log_dict({"val_loss": loss_mel}, on_epoch=True, sync_dist=True)
 
         # plot audio once per epoch
         if batch_idx == 0 and isinstance(self.logger, WandbLogger) and HAVE_WANDB:
             clips = []
             specs = []
-            for i in range(min(5, audio.shape[0])):
+            for i in range(min(self.num_log_audios, audio.shape[0])):
                 clips += [
                     wandb.Audio(
                         audio[i, : audio_len[i]].data.cpu().numpy(),
-                        caption=f"real audio {i}",
+                        caption=f"real audio {i}, epoch {self.current_epoch}",
```

Comment:
Does this produce multiple audio files on wandb? Or will it still show one audio file?

Reply:
Still show one audio file.